package.json

```diff
@@ -78,72 +78,72 @@
-		"@wordpress/router": "0.20.0",
-		"@wordpress/server-side-render": "4.28.1",
-		"@wordpress/shortcode": "3.51.0",
-		"@wordpress/style-engine": "1.34.0",
-		"@wordpress/sync": "0.13.0",
-		"@wordpress/token-list": "2.51.0",
-		"@wordpress/undo-manager": "0.11.0",
-		"@wordpress/url": "3.52.0",
-		"@wordpress/viewport": "5.28.0",
-		"@wordpress/warning": "2.51.0",
-		"@wordpress/widgets": "3.28.1",
-		"@wordpress/wordcount": "3.51.0",
+		"@wordpress/a11y": "3.51.1",
+		"@wordpress/annotations": "2.51.2",
+		"@wordpress/api-fetch": "6.48.1",
+		"@wordpress/autop": "3.51.1",
+		"@wordpress/blob": "3.51.1",
+		"@wordpress/block-directory": "4.28.2",
+		"@wordpress/block-editor": "12.19.2",
+		"@wordpress/block-library": "8.28.2",
+		"@wordpress/block-serialization-default-parser": "4.51.1",
+		"@wordpress/blocks": "12.28.2",
+		"@wordpress/commands": "0.22.2",
+		"@wordpress/components": "26.0.2",
+		"@wordpress/compose": "6.28.1",
+		"@wordpress/core-commands": "0.20.2",
+		"@wordpress/core-data": "6.28.2",
+		"@wordpress/customize-widgets": "4.28.2",
+		"@wordpress/data": "9.21.1",
+		"@wordpress/data-controls": "3.20.1",
+		"@wordpress/dataviews": "0.5.2",
+		"@wordpress/date": "4.51.1",
+		"@wordpress/deprecated": "3.51.1",
+		"@wordpress/dom": "3.51.1",
+		"@wordpress/dom-ready": "3.51.1",
+		"@wordpress/edit-post": "7.28.2",
+		"@wordpress/edit-site": "5.28.2",
+		"@wordpress/edit-widgets": "5.28.2",
+		"@wordpress/editor": "13.28.2",
+		"@wordpress/element": "5.28.1",
+		"@wordpress/escape-html": "2.51.1",
+		"@wordpress/format-library": "4.28.2",
+		"@wordpress/hooks": "3.51.1",
+		"@wordpress/html-entities": "3.51.1",
+		"@wordpress/i18n": "4.51.1",
+		"@wordpress/icons": "9.42.1",
+		"@wordpress/interactivity": "5.0.1",
+		"@wordpress/interactivity-router": "1.1.1",
+		"@wordpress/interface": "5.28.2",
+		"@wordpress/is-shallow-equal": "4.51.1",
+		"@wordpress/keyboard-shortcuts": "4.28.1",
+		"@wordpress/keycodes": "3.51.1",
+		"@wordpress/list-reusable-blocks": "4.28.2",
+		"@wordpress/media-utils": "4.42.1",
+		"@wordpress/notices": "4.19.1",
+		"@wordpress/nux": "8.13.2",
+		"@wordpress/patterns": "1.12.2",
+		"@wordpress/plugins": "6.19.2",
+		"@wordpress/preferences": "3.28.2",
+		"@wordpress/preferences-persistence": "1.43.1",
+		"@wordpress/primitives": "3.49.1",
+		"@wordpress/priority-queue": "2.51.1",
+		"@wordpress/private-apis": "0.33.1",
+		"@wordpress/redux-routine": "4.51.1",
+		"@wordpress/reusable-blocks": "4.28.2",
+		"@wordpress/rich-text": "6.28.2",
+		"@wordpress/router": "0.20.1",
+		"@wordpress/server-side-render": "4.28.2",
+		"@wordpress/shortcode": "3.51.1",
+		"@wordpress/style-engine": "1.34.1",
+		"@wordpress/sync": "0.13.1",
+		"@wordpress/token-list": "2.51.1",
+		"@wordpress/undo-manager": "0.11.1",
+		"@wordpress/url": "3.52.1",
+		"@wordpress/viewport": "5.28.1",
+		"@wordpress/warning": "2.51.1",
+		"@wordpress/widgets": "3.28.2",
+		"@wordpress/wordcount": "3.51.1",
```

Comment:
This is only updating all the packages because it's the first package release creating to wp-6.5 dist tag. So all the packages were updated even if there were no changes in all of them.